setup.py

```diff
@@ -1,9 +1,9 @@
 from distutils.core import setup, Extension
 from pip.req import parse_requirements
-install_reqs = parse_requirements('requirements.txt', session=False)
+# install_reqs = parse_requirements('requirements.txt', session=False)
```

Comment:
i'm not entirely sure about all this setup tools mess. my understanding is that the new pip *automatically* installs requirements in a `requirements.txt`, so no need to pass it here anymore. In fact, this gives errors when parsing the new directives for doing different things depending on the python version. Do we have a setuptools expert in our midst who could look at this?